baselines/bigquery-storage/src/v1beta1/big_query_storage_client.ts.baseline

```diff
@@ -198,6 +224,8 @@ export class BigQueryStorageClient {
         return apiCall(argument, callOptions, callback);
       };
     }
+
+    return this.bigQueryStorageStub;
```

Comment:
I assume `this._gaxModule.createApiCall`, does `initialize` not get called when you `new` a client?